src/components/GlobalPreferences/Network/Network.js

```diff
@@ -239,9 +239,6 @@ async function validateNetworkSettings(
     }
     if (err instanceof NoConnection) {
       settingsErrors.ethError = 'Could not connect to node'
-    } else {
-      settingsErrors.ethError = 'URI does not seem to be a ETH node'
```

Comment:
To make sure—was this intended to be removed?

Reply:
ah yes i realized that we are capturing the error with the NoConnection one